src/query/execute_schema.c

```diff
@@ -10572,7 +10572,7 @@ do_change_att_schema_only (PARSER_CONTEXT * parser, DB_CTMPL * ctemplate, PT_NOD
 
   /* the serial property has not changed, we are only dealing with renaming */
   if (is_att_prop_set (attr_chg_prop->p[P_NAME], ATT_CHG_PROPERTY_DIFF)
-      && attribute->info.attr_def.auto_increment != NULL
+      && is_att_prop_set (attr_chg_prop->p[P_AUTO_INCR], ATT_CHG_PROPERTY_PRESENT_OLD)
```

Comment:
I think this routines to change the serial name. Why should it be done when old serial is present?

Reply:
the property auto_increment has no name. the auto_increment is set once it can not be dropped.

Reply:
The serial name related to the auto_increment should be also renamed.

Reply:
I heard from kim as a routine that changes the auto increment name when the column name is changed.
I got it.